history-of-the-quakers-v3/original/14-book-4-chapter-28.adoc

```diff
@@ -169,6 +182,8 @@ Ever an advocate for a full toleration of religion,
 he established it in the first place on the broadest bottom,
 the leading article of the new constitution being this:
 
+[quote]
```

Comment:
Maybe this should be a [.embed.document.legal]?? Idk!

Reply:
ehhh... i think i like it as a quote. let's leave it. i could go either way though. :)